core/src/main/scala/com/pingcap/tispark/auth/MySQLPriv.scala

```diff
@@ -0,0 +1,111 @@
+      // ShowViewPriv is the privilege to show create view.
+  ShowViewPriv,
+      // CreateRolePriv the privilege to create a role.
+  CreateRolePriv,
+      // DropRolePriv is the privilege to drop a role.
+  DropRolePriv,
+      // CreateTMPTablePriv is the privilege to create a temporary table.
+  CreateTMPTablePriv, LockTablesPriv, CreateRoutinePriv, AlterRoutinePriv, EventPriv,
+      // ShutdownPriv the privilege to shutdown a server.
+  ShutdownPriv,
+      // ReloadPriv is the privilege to enable the use of the FLUSH statement.
+  ReloadPriv,
+      // FilePriv is the privilege to enable the use of LOAD DATA and SELECT ... INTO OUTFILE.
+  FilePriv,
+      // ConfigPriv is the privilege to enable the use SET CONFIG statements.
+  ConfigPriv,
+      // CreateTablespacePriv is the privilege to create tablespace.
+  CreateTablespacePriv,
+      // ReplicationClientPriv is used in MySQL replication
+  ReplicationClientPriv,
+      // ReplicationSlavePriv is used in MySQL replication
+  ReplicationSlavePriv,
+      // AllPriv is the privilege for all actions.
+  AllPriv, /*
+   *  Please add the new priv before AllPriv to keep the values consistent across versions.
+   */
+
+  // ExtendedPriv is used to successful parse privileges not included above.
+  // these are dynamic privileges in MySQL 8.0 and other extended privileges like LOAD FROM S3 in Aurora.
+  ExtendedPriv = Value
+
+  final val Str2Priv = Map(
+    "CREATE" -> MySQLPriv.CreatePriv,
+    "SELECT" -> MySQLPriv.SelectPriv,
+    "INSERT" -> MySQLPriv.InsertPriv,
+    "UPDATE" -> MySQLPriv.UpdatePriv,
+    "DELETE" -> MySQLPriv.DeletePriv,
+    "SHOW DATABASES" -> MySQLPriv.ShowDBPriv,
+    "SUPER" -> MySQLPriv.SuperPriv,
+    "CREATE USER" -> MySQLPriv.CreateUserPriv,
+    "CREATE TABLESPACE" -> MySQLPriv.CreateTablespacePriv,
+    "TRIGGER" -> MySQLPriv.TriggerPriv,
+    "DROP" -> MySQLPriv.DropPriv,
+    "PROCESS" -> MySQLPriv.ProcessPriv,
+    "GRANT OPTION" -> MySQLPriv.GrantPriv,
+    "REFERENCES" -> MySQLPriv.ReferencesPriv,
+    "ALTER" -> MySQLPriv.AlterPriv,
+    "EXECUTE" -> MySQLPriv.ExecutePriv,
+    "INDEX" -> MySQLPriv.IndexPriv,
+    "CREATE VIEW" -> MySQLPriv.CreateViewPriv,
+    "SHOW VIEW" -> MySQLPriv.ShowViewPriv,
+    "CREATE ROLE" -> MySQLPriv.CreateRolePriv,
+    "DROP ROLE" -> MySQLPriv.DropRolePriv,
+    "CREATE TEMPORARY TABLES" -> MySQLPriv.CreateTMPTablePriv,
+    "LOCK TABLES" -> MySQLPriv.LockTablesPriv,
+    "CREATE ROUTINE" -> MySQLPriv.CreateRoutinePriv,
+    "ALTER ROUTINE" -> MySQLPriv.AlterRoutinePriv,
+    "EVENT" -> MySQLPriv.EventPriv,
+    "SHUTDOWN" -> MySQLPriv.ShutdownPriv,
+    "RELOAD" -> MySQLPriv.ReloadPriv,
+    "FILE" -> MySQLPriv.FilePriv,
+    "CONFIG" -> MySQLPriv.ConfigPriv,
+    "USAGE" -> MySQLPriv.UsagePriv,
+    "REPLICATION CLIENT" -> MySQLPriv.ReplicationClientPriv,
+    "REPLICATION SLAVE" -> MySQLPriv.ReplicationSlavePriv,
+    "ALL PRIVILEGES" -> MySQLPriv.AllPriv)
+
+  def main(args: Array[String]): Unit = {
```

Comment:
This looks like test code that shouldn't be here.

Reply:
done